purchase_analytic/views/purchase_views.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" ?>
```

Comment:
Why this? I'd avoid this, AFAIK it's preferred to provide the encoding explicitly.